.gitlab-ci.yml

```diff
@@ -162,6 +163,7 @@ build:master_docker_latest:
     - build:master_docker_latest
   script:
     - ./test/toolbox_image.sh .build_*/um-update*.xz.img
+    - ./test/prepare_disk.sh .build_*/um-update*.xz.img
```

Comment:
for our tests, i'd recommend putting each in its own job. feel free to template it so it only reads:
```yaml
test:toolboxtest_1/2:
<<: *job_template
script: ./test/toolbox_image.sh ...
```
or something. a) it allows for the tests to run in parallel; b) you more easily see which tests fails in the overview, rather then 'all tests failed'